app/views/users/show.html.erb

```diff
@@ -1 +1,13 @@
-<h1>This should be a page where you can see each individual user's profile</h1>
\ No newline at end of file
+<div>
+  <%= render 'layouts/custom/user_card', user: @user %>
+
+  <div class="user-bio">
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </div>
+
+  <% @posts.each do |post| %>
+    <%= render 'layouts/custom/post_card', post: post %>
+  <% end %>
+  <button>See all posts</button>
```

Comment:
- [ ] Please let us make this a link that takes us to all user's post. See below:

    ```ruby
    <button>
      <%= link_to 'See all Post', user_posts_path(@user) %>
    </button>
    ```